opml-importer.php

```diff
@@ -1,13 +1,13 @@
 <?php
 /*
 Plugin Name: OPML Importer
-Plugin URI: http://wordpress.org/extend/plugins/opml-importer/
+Plugin URI: https://wordpress.org/extend/plugins/opml-importer/
```

Comment:
/extend/ is long gone, so can be removed from here :) 

```suggestion
Plugin URI: https://wordpress.org/plugins/opml-importer/
```